docs/core-plugins/console/console.md

```diff
@@ -0,0 +1,34 @@
+# Console Plugin
+
+The `Console` Plugin can store and make requests to OpenSearch directly from the user.
+
+One way you can access the `Console` plugin is to navigate to `Dev Tools`.
+
+![Console plugin](assets/console_plugin.png)
+
+### Generating specs for autocompletion
+
+The autocompletion in this section is based on files stored [here](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/console/server/lib/spec_definitions). If the REST API for OpenSearch was updated then the spec can be regenerated and stored by doing the following:
```

Comment:
nit - I'd probably just use the bare path/link to the file - it's helpful if people just want to search in their IDE, and `here` doesn't really provide any info.